src/core/lib/iomgr/ev_poll_posix.cc

```diff
@@ -108,8 +121,19 @@ struct grpc_fd {
   grpc_closure* on_done_closure;
 
   grpc_iomgr_object iomgr_object;
+
+  /* Only used by poll when GRPC_ENABLE_FORK_SUPPORT=1 */
+  grpc_fork_fd_list* fork_fd_list;
 };
 
+/* True when GRPC_ENABLE_FORK_SUPPORT=1 and polling strategy is poll. We do not
```

Comment:
remove "and polling strategy is poll". Its implicit

Reply:
done